lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-azure-cloud_manager-provision.rb

```diff
@@ -1,4 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Azure_CloudManager_Provision < MiqAeServiceManageIQ_Providers_CloudManager_Provision
+    expose :resource_group, :association => true
```

Comment:
This should be: `expose_eligible_resources :resource_group`

The `resource_group ` relationship on the provider is already exposed in [service_models/miq_ae_service_manageiq-providers-azure-cloud_manager.rb](https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-azure-cloud_manager.rb#L1)